grails-app/i18n/messages.properties

```diff
@@ -3232,6 +3238,12 @@ react.default.select.typeToSearch.label=Type to search
 react.default.noOptions.label=No options
 react.default.dateInput.placeholder.label=Select a date
 react.default.form.field.required.label=${0} is required
+
+# Date formats
+react.default.datetime.format=MM/DD/YYYY HH:mm:ss Z
+react.default.date.format=MM/DD/YYYY
+react.default.monthyear.format=MM/YYYY
```

Comment:
It's necessary because we are pulling only the labels starting with `react`... And the `react.default` are the labels we always pull.
And yes, there is a small change between the formats.